content/docs/2_cookbook/9_setup/0_caddy/cookbook-recipe.txt

```diff
@@ -0,0 +1,124 @@
+Title: Running Kirby on a Caddy v2 web server
+
+----
+
+Published: 2023-08-03
+
+----
+
+Description: How to configure the popular and simple Caddy v2 web server to run a Kirby website.
+
+----
+
+Authors:
+
+- authors/moritz-stueckler
+
+----
+
+Text:
+
+[Kirby states](https://getkirby.com/docs/guide/quickstart) that it is able to run on many different web servers. However, in reality it is mostly Apache and sometimes Nginx which are being used as webservers to run Kirby on. But there is also another, modern and very easy-to-configure alternative in the webserver world: [Caddy v2](https://caddyserver.com/v2).
```

Comment:
"Kirby states" sounds a bit like the reality is different. :D

What about this:

```suggestion
Kirby is compatible with (link: docs/guide/quickstart text: many different web servers). However, in reality it is mostly Apache and sometimes Nginx which are being used as webservers to run Kirby on. But there is also another, modern and very easy-to-configure alternative in the webserver world: [Caddy v2](https://caddyserver.com/v2).
```